apps/google-analytics-4/frontend/src/components/config-screen/NextStepsSection/NextStepsSection.tsx

```diff
@@ -0,0 +1,45 @@
+import HyperLink from 'components/common/HyperLink/HyperLink';
+import { Paragraph, Subheading } from '@contentful/f36-components';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+
+export const NO_CONTENT_TYPES_MSG =
+  'Be sure to save this configuration. Once you have configured a content type, head over to the Content tab, open an entry associated with a configured content type, and view the app in the sidebar.';
```

Comment:
maybe change "...open an entry associated with a configured content type..." to "...open an entry of a configured content type..."? Just a bit less wordy.

Reply:
Nice, I agree, the less wordy the better 💯